.venv

```diff
@@ -0,0 +1,5 @@
+# Activate: source .venv
+virtualenv .python
+source .python/bin/activate
+pip install --upgrade -e .
+pip install tox
```

Comment:
Is this a common pattern? This feels like it should belong in the README instead, maybe in a "Running the tests" subsection inside "Developer notes".